src/app/containers/Article/index.test.jsx

```diff
@@ -1,5 +1,8 @@
 import React from 'react';
```

Comment:
Im dubious of this testing file, it feels like it should stub all the `PageHandlers`, as its really testing them

Reply:
Will look at this again, but I think this should change under another issue. For now the fully rendered snapshot is effect testing the composing of the multiple HOC's.

Reply:
This has now been updated to mock each of the HOC (the page handlers) the idea behind this test is to ensure that the each HOC is called in the correct order ultimately returning the original components passed in.

The article itself is tested within ArticleMain and each HOC is tested individually as well. This is just testing the composing of the HOC.

Any further testing/refactoring will be moved to a new issue.